modules/ctrl_tcp/ctrl_tcp.c

```diff
@@ -170,8 +170,9 @@ static bool command_handler(struct mbuf *mb, void *arg)
 	struct mbuf *resp = mbuf_alloc(2048);
 	struct re_printf pf = {print_handler, resp};
 	struct odict *od = NULL;
-	const struct odict_entry *oe_cmd, *oe_prm, *oe_tok;
+	const struct odict_entry *oe_cmd, *oe_prm, *oe_dat, *oe_tok;
```

Comment:
is it possible to make a separate patch out of the "ctrl_tcp" changes ?

the patch is already quite large and difficult to review

Reply:
ok.

Reply:
Same here. Change in ctrl_tcp will be obsolete.